### PR TITLE
add support for appendSubheadersToContainer option

### PR DESCRIPTION
**Pull request by bcherny**

### CODE REVIEW

**Comment by albertywu** [comment]
Just so I understand, we now support an option to append the subHeaders container to the main grid container (on non-pinned grids) via a grid option. Is this correct?

Could you update the README?

File: slick.grid.js [hunk 1]
```diff
@@ -347,14 +348,21 @@ if (typeof Slick === "undefined") {
...
-      topCanvas.el[0].appendChild(subHeaders.el[0]);
-      topCanvas.el[1].appendChild(subHeaders.el[1]);
       topViewport.el[0].appendChild(topCanvas.el[0]);
       topViewport.el[1].appendChild(topCanvas.el[1]);
       contentViewport.el[0].appendChild(contentCanvas.el[0]);
       contentViewport.el[1].appendChild(contentCanvas.el[1]);
       contentViewportWrap.el[0].appendChild(contentViewport.el[0]);
       contentViewportWrap.el[1].appendChild(contentViewport.el[1]);
+
+      // TODO: support subHeaders.el[1]
```
Comment (albertywu) [inline slick.grid.js]: does this mean we can't use this feature on a pinned grid? What is the extra work effort to support pinned?

**Comment by albertywu** [comment]
Not sure I totally understand, but LGTM

File: slick.grid.js (hunk 1 above)
Comment (bcherny) [inline slick.grid.js]: for now, no. if we ahve a use case in the future, we'll have to inject the subheaders into a container  with left/right subcontainers and size them appropriately, just like what we already do for the header and body. punting for now.

**Comment by bcherny** [comment]
updated readme

**Pull request merged**